src/main/java/summaryPackage/RouteCalculation.java

```diff
@@ -0,0 +1,61 @@
+package summaryPackage;
+
+import calculation.CalculateCoordinates;
+import calculation.CalculateP2PDistance;
+import cli.CliProperties;
+import http.Client;
+import io.restassured.response.Response;
+import lombok.extern.slf4j.Slf4j;
+import model.LocationPoint;
+import model.Route;
+import reader.DataReader;
+import reader.RouteSerializer;
+import utils.ResponseUtils;
+import utils.Utilities;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static utils.ResponseUtils.validateJsonAgainstSchema;
+
+@Slf4j
+public class RouteCalculation {
+
+  public List<Route> newPoints(CliProperties cliProperties) {
```

Comment:
please try to name methods using verbs, like calculateNewPoints, or just calculatePoints